packages/cli/test/commands/pull.test.ts

```diff
@@ -41,5 +55,78 @@ describe('pull', () => {
     expect(exitCode).toEqual(0);
     expect(actualEnv).toBeTruthy();
     expect(raw.includes('# Created by Vercel CLI')).toBeTruthy();
+
+    // --env flag does not affect nested (under .vercel) files
+    const nestedEnvFileExists = await fs.pathExists(
+      path.join(cwd, '.vercel', '.env.development.local')
+    );
+    expect(nestedEnvFileExists).toBeTruthy();
```

Comment:
Should this test compare the contents instead of checking existence? Its not clear what "does not affect" means here.

Reply:
I updated the comment to say that the file name is not affected. I also tested the contents to make sure it at least has the env file header line.